releasenotes/notes/usm-postgres-metrics-messages-count-54cd26ca83cc77d7.yaml

```diff
@@ -0,0 +1,3 @@
+enhancements:
+  - |
+    Additional metrics about the distribution of the number of Postgres messages in each TCP packet.
```

Comment:
```suggestion
    Additional metrics about the distribution of the number of Postgres messages in each TCP packet have been added.
```

Reply:
I have paraphrased the description of the improvement, these metrics are being added for the first time for Postgres. Thank you.

Reply:
@yuri-lipnesh please remove the changelog entry, and add `no-changelog` tag of the PR

Reply:
I've addedd appropriate tags

Reply:
but the releasenote still exist, please remove it

Reply:
done